WordPressKit/WordPressKit/PluginDirectoryServiceRemote.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import Alamofire
+
+public struct PluginDirectoryGetInformationEndpoint: Endpoint {
+    static let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "YYYY-MM-dd h:mma z"
```

Comment:
I came accros this article today: https://app-o-mat.com/post/why-why-why-why mentioning the usage of `YYYY` vs `yyyy`, could it be an issue here? :)